services/app-api/src/resolvers/questionResponse/createContractQuestion.test.ts

```diff
@@ -101,10 +81,17 @@ describe('createQuestion', () => {
             ],
         })
 
-        const indexQuestionsPayload = await indexTestQuestions(
-            cmsServer,
-            submittedPkg.id
-        )
+        const contractWithQuestions = await stateServer.executeOperation({
```

Comment:
@JasonLin0991 Can we get a test helper for this 🙏 like the other api requests . Let me know if you want me to make this change since I know you moved on to other work, I have bandwidth.